src/client/apiclient.ts

```diff
@@ -1,7 +1,8 @@
 import axios, { AxiosResponse, AxiosError, Method, AxiosRequestConfig } from "axios";
-import {ALPHABETICAL_SEARCH_URL} from "../config/config";
+import {ALPHABETICAL_SEARCH_URL, AUTH_KEY} from "../config/config";
+import Cookies = require("cookies");
 
-const GET_COMPANIES_PATH: string = ALPHABETICAL_SEARCH_URL;
+const GET_COMPANIES_PATH: string = "http://" + ALPHABETICAL_SEARCH_URL;
```

Comment:
I would include the protocol in the ALPHABETICAL_SEARCH_URL variable, because in live it could be served over https